examples/basic-express-example/README.md

```diff
@@ -0,0 +1,20 @@
+## Sync sample application example
+
+## Requirements
+
+- mongodb 
+
+MONGO_CONNECTION_URL environment variable needs to point to mongodb instance
+
+- redis
+
+REDIS_CONNECTION_URL  environment variable needs to point to running redis instance
```

Comment:
So these are being defaulted to localhost with the default ports, which can make, so maybe just change a little bit of the text here to point that out.